<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="fileName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectImage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageFilePreview">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>19</y>
     <width>561</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>Select an Image for Extraction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="dataExtract">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>430</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
